security-utils/src/reqwest.rs

```diff
@@ -7,6 +7,8 @@ use color_eyre::{
 use hex_literal::hex;
 use reqwest::{Certificate, Client, ClientBuilder};
 
+pub use reqwest;
+
```

Comment:
should this line be part of the PR?

Reply:
Yes, it re-exports the reqwest crate. This can be useful to use the same crate version across dependencies.